packages/sources/tradingeconomics-test/src/config/index.ts

```diff
@@ -0,0 +1,28 @@
+export const defaultEndpoint = 'price'
+export const customSettings = {
+  API_ENDPOINT: {
+    description: 'The HTTP URL to retrieve data from',
+    type: 'string',
+    default: 'https://api.tradingeconomics.com/markets',
+    required: true,
+  },
+  WS_API_ENDPOINT: {
+    description: 'The WS URL to retrieve data from',
+    type: 'string',
+    default: 'wss://stream.tradingeconomics.com/',
+    required: true,
+  },
+  API_CLIENT_KEY: {
+    description: 'The TradingEconomics API client key',
+    type: 'string',
+  },
```

Comment:
Based on the v2 adapter I think this should be marked as required
```suggestion
  API_CLIENT_KEY: {
    description: 'The TradingEconomics API client key',
    type: 'string',
    required: true,
  },
```